marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/flow/testTargetCollectionAdditivity/flowWithTargetCollectionsAdditivity.sjs

```diff
@@ -0,0 +1,74 @@
+const DataHubSingleton = require("/data-hub/5/datahub-singleton.sjs");
```

Comment:
This test will be a lot simpler by using flowRunner.sjs . With processContentWithFlow, you can pass in your own contentArray, which means you can set content.context.collections to whatever you want - no need to persist anything. So you can do something like this:

```
const content = [{"uri":"whatever", "value":{"hello":"world"}, context: {collections: ["one", "two"]}];
flowRunner.processContentWithFlow("simpleMappingFlow", content, jobId, options);
```

Then use hubTest.getRecord(uri) to get an object that has the document and collections in it. Much easier to test with that.